embeddings/pipeline/hugging_face_classification.py

```diff
@@ -27,13 +35,35 @@ def __init__(
         input_column_name: str,
         target_column_name: str,
         output_path: T_path,
+        document_embedding_cls: Union[str, Type[DocumentEmbedding]] = FlairDocumentPoolEmbedding,
+        sample_missing_splits: Optional[Tuple[Optional[float], Optional[float]]] = None,
+        seed: int = 441,
         task_model_kwargs: Optional[Dict[str, Any]] = None,
         task_train_kwargs: Optional[Dict[str, Any]] = None,
+        load_model_kwargs: Optional[Dict[str, Any]] = None,
+        load_dataset_kwargs: Optional[Dict[str, Any]] = None,
     ):
-        dataset = HuggingFaceDataset(dataset_name)
+
+        load_model_kwargs = {} if load_model_kwargs is None else load_model_kwargs
```

Comment:
add method/function to instantiate kwargs if None is passed

Reply:
What would be benefits of such a function?

Reply:
reduce code duplication

Reply:
but it's one liner

Reply:
ok then we should unify usage of it because we use it like this right now
`
load_model_kwargs = {} if load_model_kwargs is None else load_model_kwargs
dataset = HuggingFaceDataset(
    dataset_name, **load_dataset_kwargs if load_dataset_kwargs else {}
)
embedding = AutoFlairDocumentPoolEmbedding.from_hub(
    repo_id=embedding_name,
    document_embedding_cls=document_embedding_cls,
    **load_model_kwargs
)
`

Reply:
in huggingface_classification pipeline file and huggingface_pair_classification pipeline

Reply:
Fixed

Reply:
Fixed